superset/connectors/sqla/models.py

```diff
@@ -1180,7 +1184,7 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
             for selected in columns:
                 select_exprs.append(
                     columns_by_name[selected].get_sqla_col()
-                    if selected in columns_by_name
+                    if selected in columns_by_name and validate_adhoc_subquery(selected)
```

Comment:
This and line 1395 would also require some modification. Here something like:

```python
for selected in columns:
    validate_adhoc_subquery(selected)
    selec_exprs.append(...)
```

Reply:
that makes sense. thank you for clarifying your comments. I will update the method accordingly 😄.